.env.dev

```diff
@@ -8,7 +8,7 @@ FIREBASE_APP_ID=1:2298664147:web:340bda75cd5d25f3997026
 FIREBASE_MEASUREMENT_ID=G-B7GDQ1Y9LL
 <!-- firebase configs ends -->
 API_ENDPOINT=https://tfr.dev.thefittingroom.xyz/v1
-LANGUAGE_URL=https://assets.dev.thefittingroom.xyz/shop-sdk/4200127/languages
+LANGUAGE_URL=https://assets.dev.thefittingroom.xyz/shop-sdk/${GITHUB_SHA::7}/languages
```

Comment:
Wouldn't this break it when running locally? I don't think we'd want to make running locally more difficult.

Reply:
when we run locally it uses the .env instead of .env.dev i think it should be fine